driver-async/src/main/com/mongodb/async/client/MongoClientOptions.java

```diff
@@ -231,17 +245,6 @@ public MongoClientOptions build() {
     }
 
     /**
-     * Gets the description for this MongoClient, which is used in various places like logging and JMX.
-     *
-     * <p>Default is null.</p>
-     *
-     * @return the description
-     */
-    public String getDescription() {
```

Comment:
This was part of the Options clean up - its useless and not used.  The real description that is needed and used is hiding in ClusterSettings that has to be provided to Options.